README-developers.md

```diff
@@ -185,3 +185,167 @@ username = your_test_pypi_usename
 password = your_test_pypi_password
 ```
 Be sure to make this file publicly unreadable since it contains your passwords.
+
+## USING GITHUB TO CREATE A DISTRIBUTION
+
+### One-time steps
+
+#### Create an account on PyPI and Test PyPI
+
+Follow the steps for creating a new account at [Test PyPI](https://test.pypi.org/)
+and at [PyPi](https://pypi.org/)
+
+#### Get Permission to access cspyce
+
+Separate permission needs to be provided for both PyPI and Test PyPI.
+For now, only Frank and Rob can do this.
+They should visit these two URLs and add you as a collaborator.
+- https://pypi.org/manage/project/cspyce/collaboration/
+- https://test.pypi.org/manage/project/cspyce/collaboration
+
+
+#### Create tokens for PyPI and Test PyPI
+
+PyPI and Test PyPI have separate API tokens.
+
+Visit each of the following two URLs in turn:
+  - https://pypi.org/manage/account/token/
+  - https://test.pypi.org/manage/account/token
+
+For each of the URLs, 
+1. Fill in the two fields. The token name can be whatever you want.
+The scope should be `Project: cspyce`
+2. Click "Add token"
+3. Keep a copy of the text that is generated. This is your API Token.
+
+
+####  (Optional) Update your `.pypirc` file.
+
+You can use the tokens created in the previous step in your `~/.pypirc` configuration
+file rather than a username and password.
+Use `__token__` as the username and the appropriate token generated above as the
+password.
+
+#### Tell GitHub these secrets.
+
+1. Log into your github repository for pds-cspyce
+2. Click "settings", then "secrets" on the left-hand menu, then "actions".
+3. Use the "New Repository Secret" button to add two secrets named `PYPI_API_TOKEN` 
+and `TEST_PYPI_API_TOKEN`. 
+The value of each of these two secrets should be the appropriate API token
+generated above.
+
+### Creating a new distribution.
+
+#### Step 1: Update the version
+
+When you are planning on creating a pull request that requires a new distribution, make
+sure that you update the version number in `setup.py`.
+The version number appears in the `do_setup()` function at the very end of the file.
+
+> Note to Rob: When approving a request that will require making a new distribution, 
+> ensure that the version number is updated.
+> 
+#### Step 2: Create a new branch
+
+Make a new branch that is a clone of the pull request. 
+I've been calling my clone `GitActions` (and will use that name in the rest of this
+document).
+
+Do the following:
+
+  1. You should have a local copy of the pull request.
```

Comment:
A weird use of the term "pull request". I think you mean you should have a local copy of the branch that is referenced in the pull request. Although there doesn't need to be a pull request to create a distribution, so why even talk about pull requests? Just say you need to have an up-to-date copy of the repo and have the appropriate development branch checked out by doing `git pull` and `git checkout <branchname>`.